src/main/java/org/intellij/sdk/codesync/ui/dialogs/PricingAlertDialog.java

```diff
@@ -27,10 +28,27 @@ public PricingAlertDialog(Boolean isOrgRepo, Boolean canAvailTrial, String prici
         this.pricingURL = pricingURL;
 
         if (canAvailTrial) {
-            upgradeButtonText = "Try Pro for free";
-            secondaryMessage = isOrgRepo ? Notification.TRY_ORG_PRO_FOR_FREE: Notification.TRY_PRO_FOR_FREE;
+            upgradeButtonText = isOrgRepo ? NotificationButton.TRY_TEAM_FOR_FREE: NotificationButton.TRY_PRO_FOR_FREE;
+            secondaryMessage = isOrgRepo ? Notification.TRY_TEAM_PLAN_FOR_FREE: Notification.TRY_PRO_FOR_FREE;
         } else {
-            upgradeButtonText = "Upgrade to Pro";
+            upgradeButtonText = NotificationButton.UPGRADE_PLAN;
+            secondaryMessage = isOrgRepo ? Notification.UPGRADE_ORG_PRICING_PLAN: Notification.UPGRADE_PRICING_PLAN;
+        }
+
+        setTitle(this.title);
+    }
+
+    public PricingAlertDialog(Boolean isOrgRepo, Boolean canAvailTrial, String pricingURL, Project project, boolean privateRepoCountLimitReached){
```

Comment:
We probably can keep only 1 param here `boolean privateRepoCountLimitReached` and then can above overloaded method with 
1. isOrgRepo = false
2. canAvailTrial will come from the API
3. pricingURL will come from the constants